2-mandatory/3-stocks.js

```diff
@@ -48,9 +56,18 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
```

Comment:
One question: What happens if we have more than 5 prices for each stock? Will the code still work?
If not, what can you do to make it work for any number of prices?

Reply:
> One question: What happens if we have more than 5 prices for each stock? Will the code still work? If not, what can you do to make it work for any number of prices?

I did solve the code and change 5 to closingPricesForAllStocks.length 
also push after editing